app/src/main/res/layout/item_currency.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:background="?android:attr/selectableItemBackground"
+    android:layout_width="match_parent"
+    android:layout_height="56dp"
+    android:gravity="center_vertical"
+    android:orientation="horizontal"
+    android:paddingEnd="24dp"
+    android:paddingStart="24dp">
+
+    <FrameLayout
+        android:id="@+id/currency_symbol"
+        android:layout_width="20dp"
+        android:layout_height="20dp">
+
+        <TextView
+            android:id="@+id/symbol_text"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:background="@drawable/bg_coin_symbol_white"
+            android:gravity="center"
+            android:maxLines="1"
+            android:textColor="@color/gray_50"
+            android:textStyle="bold"
+            tools:ignore="MissingPrefix"
```

Comment:
could you not use this?

Reply:
I don't understand what about you are telling me

Reply:
tools:ignore="MissingPrefix"  do you know why you put it here? if you use it you should know why